multigeiger/speaker.cpp

```diff
@@ -156,6 +163,33 @@ void tick_enable(bool enable) {
   }
 }
 
+void IRAM_ATTR alarm() {
```

Comment:
why is this IRAM_ATTR?

Reply:
Ah, thought that anything that plays with the &mux_audio must be quickly accessible from IRAM. But e.g. `play` is not, so probably not necessary. Removed in next commit.